src/Search/Microsoft.Azure.Search/GeneratedSearchService/Models/ClassicTokenizer.cs

```diff
@@ -40,8 +40,10 @@ public ClassicTokenizer(string name, int? maxTokenLength = default(int?))
         }
 
         /// <summary>
-        /// Gets or sets the maximum token length. Tokens longer than the
-        /// maximum length are split. Default is 255.
+        /// Gets or sets the maximum token length. Default is 255. Tokens
+        /// longer than the maximum length are split. The maximum token
+        /// length that can be used is 300 characters. Tokens longer than 300
```

Comment:
It's not true that tokens longer than 300 will be ignored, I've just checked. Please change the message to : 

Gets or sets the maximum token length. Default is 255. Tokens longer than the maximum length are split. The maximum token length that can be used is 300 characters.